go-controller/pkg/factory/factory.go

```diff
@@ -501,6 +503,13 @@ func (wf *WatchFactory) GetPods(namespace string) ([]*kapi.Pod, error) {
 	return podLister.Pods(namespace).List(labels.Everything())
 }
 
+// GetPods returns all the pods in a given namespace
```

Comment:
`GetPods returns all the pods in a given namespace` -> `GetPodsWithSelector returns a list of pods matching the selector`

Reply:
Fixed, Thanks!